CiviWiki/OpenCiviWiki
# Wassaf/1024 ci  automating testing

What is the relevant  ticket ? 
Closes #1024 

What does this PR do ? 

This PR has the following changes
   - Removes unused code.
   - Fixes broken imports
   - Adds workflow to use pytest

How can this PR be manually tested?

This PR can be tested in the following way
- Update code and push.
- Verify test run in `Action` tab.
